server/app/models/faq.rb

```diff
@@ -0,0 +1,6 @@
+class Faq < ApplicationRecord
+
+    validates :question, presence: true, length: { minimum: 10 }
+    validates :answer, presence: true, length: { minimum: 2 }
+
```

Comment:
Could we do `Content` with attributes `title`, `content`, for future proofing? I think it can help expanding on this model and not add a model for every type of content.

Reply:
Yeah the naming of the whole model. The name doesn't have to be exact but I think having a more generalizable model is better. If the time is too tight, maybe change it at the beginning of the next iteration?

Reply:
Yeah. I could change it at the beginning of next iteration.